src/app/servicesModule/services/create-poll-storage.service.ts

```diff
@@ -51,10 +51,18 @@ export class CreatePollStorageService {
     const connectionConfig = ConnectionConfig.createWithLocalIpfsConnection(
       new BlockchainNetworkConnection(blockChainNetworkType, blockChainHost, blockChainPort, blockChainProtocol),
       new IpfsConnection(storageHost, storagePort, storageOptions));
-
-    this.uploader = new Uploader(connectionConfig);
+    const storageHostUploader = environment.storageConnectionUnload.host;
+    const storagePortUploader = environment.storageConnectionUnload.port;
+    const storageOptionsUploader = environment.storageConnectionUnload.options;
+    const connectionConfigUploader = ConnectionConfig.createWithLocalIpfsConnection(
+      new BlockchainNetworkConnection(blockChainNetworkType, blockChainHost, blockChainPort, blockChainProtocol),
```

Comment:
Use proper camelcasing for blockchain, e.i: instead of blockChain, it should be `blockchainNetworkType` etc.